src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -182,7 +183,22 @@ export default {
 			showBadge: false
 		};
 	},
+	mixins: [
+		loanChannelQueryMapMixin
+	],
 	watch: {
+		selectedCategory(category) {
```

Comment:
(same with this)

Reply:
Generally, I'm ok with in-progress work that doesn't affect any live prod code. However, this file is actively used in prod so we'll need to wait on merging until the code in this is cleaned up and verfiied in the context of the existing/running quick filters exp.

Reply:
I tested this category quick filter locally, and it doesn't currently change the displayed loans, so I would exclude this file from this PR (or complete the filter functionality).

Reply:
sounds good i can remove this!